### ISSUE TITLE
Added a new profile, existing profile dissapeared

### ISSUE THREAD

**Issue by phw** [gnunn1/terminix#294]
This is a bit strange, because I cannot reproduce it, but it definitely happened. I had only one default profile and all my terminals and saved sessions were configured to use it. I went in settings, added a second profile and then wanted to switch some open terminals to use this profile. I then realized that the old profile was missing in the list, the existing terminals had no profile selected. I can tell that I did not edit the existing profile by accident since the ID changed and it was unselected for the existing terminals.

I just wanted to report this now, but this is probably difficult to act upon until somebody is able to reproduce it. I'll try.

**Comment by gnunn1** [comment]
If you open dconf-editor and go to com/gexperts/Terminix/profiles do you see the profile there? There is a list key which is how Terminix finds the profile and then folders for each folder. Just wondering if something happened to the list key because it needs to be updated when you add a new profile.

**Comment by phw** [comment]
Indeed the old profile is still in dconf, but does not show up in Terminix. No idea why, Terminix does not show any error on startup. I have attached a screenshot of the profile in dconf:

![bildschirmfoto vom 2016-05-06 11-31-59](https://cloud.githubusercontent.com/assets/29852/15069525/76b29ebe-137e-11e6-900c-2572e882a080.png)

Is it possible that this lost profile also causes #302 somehow or is this unrelated?

**Comment by phw** [comment]
Ok, now I understood the list key thing :) No, that profile is not in the list key, that shows just `com.gexperts.Terminix.ProfilesList list ['2e3874bf-72ac-4e48-8016-54c94fdfc22f', '898e686d-6cf3-4809-a172-8dc569948a5a']`

**Comment by phw** [comment]
I think I found the issue. The ID of my profile above (the one that got removed) is `2b7c4080-0ddd-46c5-8f23-563fd3ba789d`, that is the hard coded default ID of the first profile created by terminix.

When adding a new profile this gets removed, see https://github.com/gnunn1/terminix/blob/82ea3a78cc2f449112158a250a2d675f1a6110bd/source/gx/terminix/preferences.d#L206

What exactly was the idea behind this code? From what I can see it should be safe to drop this. Otherwise it leads to the pre-configured default profile to always get lost once you add a new one.
